src/js/apps/patients/worklist/worklist_state.js

```diff
@@ -107,35 +110,66 @@ export default Backbone.Model.extend({
       [dateType]: this.formatDateRange(relativeRange, unit, dateFormat),
     };
   },
+  getDefaultStatesFilter() {
+    const isDoneOnly = this.id === 'done-last-thirty-days';
+    const { done, notDone } = this.states.groupByDone();
+
```

Comment:
I _think_ we could hard cache this.. like 
```js
if (!this._defaultStatesFilter) {
  ....
  this._defaultStatesFilter = isDoneOnly ? done : notDone;
}
return this._defaultStatesFilter;

Reply:
Couldn't get this to work... `this._defaultStatesFilter` is being reset each time the filter is changed and the worklist app restarts...